aten/src/ATen/TensorImpl.cpp

```diff
@@ -2,6 +2,9 @@
 
 #include <ATen/Tensor.h>
 #include <ATen/optional.h>
+#include <ATen/Scalar.h>
```

Comment:
why is this necessary?

Reply:
Not necessary, will remove in the mega-PR at #9690.